control/autoware_trajectory_follower_node/README.md

```diff
@@ -140,14 +140,7 @@ Giving the longitudinal controller information about steer convergence allows it
 
 #### Parameter
 
-- `ctrl_period`: control commands publishing period
-- `timeout_thr_sec`: duration in second after which input messages are discarded.
-  - Each time the node receives lateral and longitudinal commands from each controller, it publishes an `Control` if the following two conditions are met.
-    1. Both commands have been received.
-    2. The last received commands are not older than defined by `timeout_thr_sec`.
-- `lateral_controller_mode`: `mpc` or `pure_pursuit`
-  - (currently there is only `PID` for longitudinal controller)
-- `enable_control_cmd_horizon_pub`: publish `ControlHorizon` or not (default: false)
+{{ json_to_markdown("control/autoware_trajectory_follower_node/schema/simple_trajectory_follower.schema.json") }}
```

Comment:
This should be 
```suggestion
{{ json_to_markdown("control/autoware_trajectory_follower_node/schema/trajectory_follower_node.schema.json") }}
```

Reply:
We also would like to keep the detailed explanation about timeout_thr_sec parameter.